addons/website_sale_delivery/views/website_sale_delivery_templates.xml

```diff
@@ -31,7 +31,9 @@
             <div t-if="delivery_nb == 1" id="delivery_carrier" class="mt4">
                 <b>Shipping Method: </b>
                 <t t-foreach="deliveries" t-as="delivery">
-                    <t t-call="website_sale_delivery.payment_delivery_methods"/>
+                    <div class="o_delivery_carrier_select" style="display: inline;">
```

Comment:
Cannot we use the `d-inline` class instead of `display: inline;` style. 🤔 
```suggestion
                    <div class="o_delivery_carrier_select d-inline">
```

To check because I don't know if we have the class in that template. 🙂